Cython/Utility/ModuleSetupCode.c

```diff
@@ -1047,13 +1050,26 @@ static CYTHON_INLINE PyObject * __Pyx_PyDict_GetItemStrWithError(PyObject *dict,
   #define __Pyx_SET_SIZE(obj, size) Py_SIZE(obj) = (size)
 #endif
 
-#if CYTHON_COMPILING_IN_CPYTHON || CYTHON_COMPILING_IN_LIMITED_API
-#define __Pyx_REFCNT_MAY_BE_SHARED(o)  (Py_REFCNT(o) > 1)
+enum __Pyx_ReferenceSharing {
+  __Pyx_ReferenceSharing_DefinitelyUnique, // We created it so we know it's unshared - no need to check
```

Comment:
DefinitelyUnique was something that was discussed when I originally added the `REFCNT==1` checks to the indexing. It isn't currently used, but I added it to the enum so it'd be easy to add in future if we want.